w.lua

```diff
@@ -96,8 +96,8 @@ function init(jua)
       local id = findID(url)
       if id and callbackRegistry[id].closed then
         callbackRegistry[id].closed(id)
+        table.remove(callbackRegistry, id)
       end
-      table.remove(callbackRegistry, id)
```

Comment:
Been a while since I looked at this code, but this is probably more correct:
```lua
if id then
  if callbackRegistry[id].closed then
    callbackRegistry[id].closed(id)
  end
  table.remove(callbackRegistry, id)
end
```

Reply:
fixed, thanks!